tools/update-unicode-utils.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @author Toru Nagashima <https://github.com/mysticatea>
+ */
+"use strict";
+
+const fs = require("fs");
```

Comment:
Could this also be done with unicode property escapes?

```js
for (let charCode = 0; charCode < 2 ** 16; charCode++) {
    if (/\p{Mn}|\p{Mc}|\p{Me}/u.test(String.fromCharCode(charCode))) {
        combiningChars.add(charCode);
    }
}
```

It might be simpler than downloading a file from a server, although it would prevent people from running the script unless they use Node 10.

Reply:
Wow, good idea!